src/sonic-config-engine/minigraph.py

```diff
@@ -237,10 +239,27 @@ def parse_dpg(dpg, hname):
                 vlan_attributes.update(addrtuple)
                 vlan_intfs.append(copy.deepcopy(vlan_attributes))
             vlans[vintfname] = vlan_attributes
-
-
-        return intfs, lo_intfs, mgmt_intf, vlan_intfs, pc_intfs, vlans, pcs
-    return None, None, None, None, None
+        aclintfs = child.find(str(QName(ns, "AclInterfaces")))
+        acls = {}
+        for aclintf in aclintfs.findall(str(QName(ns, "AclInterface"))):
+            aclname = aclintf.find(str(QName(ns, "InAcl"))).text
+            aclattach = aclintf.find(str(QName(ns, "AttachTo"))).text.split(';')
+            acl_intfs = []
+            for member in aclattach:
+                member = member.strip()
+                if port_alias_map.has_key(member):
+                    member = port_alias_map[member]
+                if pcs.has_key(member):
+                    acl_intfs.extend(pcs[member]['members'])  # For ACL attaching to port channels, we break them into port channel members
+                elif vlans.has_key(member):
+                    print >> sys.stderr, "Warning: ACL "+aclname+" is attached to a Vlan interface, which is currently not supported"
```

Comment:
If called in systemd systemd will log it into syslog, and if called in containers docker will log it into docker logs.
What's the use case of calling this from ansible?